knexfile.ts

```diff
@@ -0,0 +1,28 @@
+import { Knex } from "knex";
+import path from "path";
+import dotenv from "dotenv";
+
+dotenv.config();
+
+const knexConfig: Knex.Config = {
+  client: "pg",
+  connection: {
+    host: process.env.DB_HOST,
```

Comment:
Can we provide a .env.example to this PR?

Reply:
Pushing now!